app/services/gibbon_service/list_service.rb

```diff
@@ -55,17 +59,18 @@ def unsubscribe_members(unsubscribable_uids = [])
       members_batches.each do |members_batch|
         p "Starting unsubscribe on mailchimp for members with uids #{members_batch.join('-')}"
         members_batch.each do |uid|
-          response = gibbon.lists(@list_uid).members(uid).update(body: { status: MEMBER_STATUS[:unsubscribe] })
+          response = gibbon.lists(@list_uid).members(uid).update(body: {status: MEMBER_STATUS[:unsubscribe]}).body
           p response
         end
         p "Finished unsubscribe on mailchimp for members with uids #{members_batch.join(', ')}"
       end
     end
 
     private
-      def retrieve_members
-        p "retrieving members for list #{@list_uid}"
-        @members = gibbon.lists(@list_uid).members.retrieve(DEFAULT_MEMBER_RETRIEVAL_PARAMS)['members']
-      end
+
+    def retrieve_members
+      p "retrieving members for list #{@list_uid}"
+      @members = gibbon.lists(@list_uid).members.retrieve(DEFAULT_MEMBER_RETRIEVAL_PARAMS)['members']
```

Comment:
@SergeyMell Here `.body` has not been used. Is there any specific reason for it?

Reply:
It's your original code. Just moved a couple of spaces to the left. But `body` should be there either. I'll add this.